Adding the certificate to Replicated for 2.x server customer (#5545)
circleci/circleci-docs: jekyll/_cci2/certificates.adoc

. Click "Save" at the bottom of the settings page and restart when prompted

Ensure the hostname is properly configured from the Management Console (`<circleci-hostname>.com:8800`) **and** that the hostname used matches the DNS records associated with the TLS certificates.

Make sure the Auth Callback URL in GitHub/GHE matches the domain name pointing to the Services machine, including the protocol used, for example `**https**://info-tech.io/`.

=== Adding the certificate to Replicated

>  Replicated Management Console(`<circleci-hostname>.com:8800`) runs on different web server, so you need to apply the above certificate for Replicated as well.

Reviewer: ```suggestion
The Replicated Management Console(`<circleci-hostname>.com:8800`) runs on a different web server, so you also need to apply the certificate used above for Replicated. To do so, follow these steps:
```

> . To do so, navigate to `https://hostname:8800/console/settings#tls-key-cert`

Reviewer: ```suggestion
. Navigate to `https://hostname:8800/console/settings#tls-key-cert`.
```

> . To do so, navigate to `https://hostname:8800/console/settings#tls-key-cert`

Reviewer: Should this be `<circleci-hostname>.com:8800` as above?

Author: @rosieyohannan Yes, and there is one more mistake for the url. Basically, hostname includes `.com` or other domain string. So the `<circleci-hostname>.com:8800` name should replace to `<circleci-hostname>:8800`. I'll send a new commit to replace all of them.

> . Upload your newly generated certificate and key

Reviewer: ```suggestion
. Upload your newly generated certificate and key.
```

> . Click "Save" at the bottom of the page and restart the Replicated UI when prompted

Reviewer: ```suggestion
. Click **Save** at the bottom of the page and restart the Replicated UI when prompted.
```

> The Replicated Management Console(`<circleci-hostname>:8800`) runs on a different web server, so you also need to apply the certificate used above for Replicated. To do so, follow these steps:

Reviewer: ```suggestion
The Replicated Management Console (`<circleci-hostname>:8800`) runs on a different web server, so you also need to apply the certificate used above for Replicated. To do so, follow these steps:
```

> . Navigate to `https://hostname:8800/console/settings#tls-key-cert`

Reviewer: ```suggestion
. Navigate to `<circleci-hostname>:8800/console/settings#tls-key-cert`
```

> Even if you can't access the Replicated Page with the hostname, you still can access by IP address.

Reviewer: @ganezasan just wondering, is this note in here because it often doesn't work? Just thinking we could make this a note so:

**Note:** Sometimes you will be unable to access the Replicated settings page listed above using the hostname. If this is the case you will be able to access using the IP address.

Author: @rosieyohannan Sorry, I forgot to add the condition of this. This means, when the TLS certification has expired, you can't access the Replicated UI with the hostname(domain name), but you still can access it by the IP address. So they can update the TLS certification on the UI.

> Also, as alternative option, you can upload the certificate and key by following command on the ServiceBox.

Reviewer: ```suggestion
Alternatively, you can upload the certificate and key by running the following command on the Services machine, substituting your hostname, path to key and path to certificate:
```

Reviewer: Is <domain> here the hostname?

Author: Yes, in this context, I should use `<circleci-hostname>`. So, I'll update the following script.

> Alternatively, you can upload the certificate and key by running the following command on the Services machine, substituting your hostname, path to key and path to certificate:

Reviewer: ```suggestion
Alternatively, you can upload the certificate and key by running the following command on the Services machine, substituting your hostname, path-to-key and path-to-certificate:
```
